summary_workflows/quantification/match_with_gt.py

```diff
@@ -0,0 +1,133 @@
+import subprocess
+from io import StringIO
+import pandas as pd
+import numpy as np
+from shutil import which
+import argparse
+
+parser = argparse.ArgumentParser()
+
+parser.add_argument('a', help='The BED file containing predictions. MUST be BED6 format.')
+parser.add_argument('b', help='The ground truth bed file. First 6 columns must be standard BED6, but can have additional columns appended.')
+parser.add_argument('window', help='Number of bases to append to each side of the predicted site.', type=int)
+#parser.add_argument('-o', help='output file directory') # not yet implemented!!
+args = parser.parse_args()
+
+
+f_GT = args.b
+f_PD = args.a
+window = args.window
+
+
+def bedtools_window(bed1, bed2, window, reverse=False):
```

Comment:
Could you explain the parameter reverse? According to bedtools the parameter "-v" means "Only report those entries in A that have no overlaps with B.", is this correct?

Reply:
Yes, this is to find the polyA sites that do not have an overlap in the ground truth set.